test/ext/disable-element.js

```diff
@@ -59,4 +59,37 @@ describe("disable-element extension", function() {
         btn2.disabled.should.equal(false);
     });
 
+    it('disables multiple elements during htmx request', function () {
+		// GIVEN:
+		// - A button triggering an htmx request with disable-element extension
+		// - Another button that needs to be disabled during the htmx request
+		// - A control button that won't be disabled during the htmx request
+		// - All buttons are enabled
```

Comment:
I appreciate the comments but I think the test is self-explanatory and it would be better to remove them :)

Reply:
da89d61